nata/simpleBlogHomePage/styles.css

```
@@ -0,0 +1,86 @@
+html {
+    margin: 0 auto;
+}
+
```

How about:

```suggestion
body {
  margin: 8px 10% 0 10%;
}
```

This will position the whole content in the center of a page